src/components/staffing/StaffingTable.js

```diff
@@ -33,7 +35,7 @@ export default class StaffingTable extends React.Component {
               field="name"
             />
           }
-          width={120}
+          width={200}
```

Comment:
I'm worried that the page is already not very responsive (does not handle well different screen sizes), maybe we could use an ellipsis when names are too long?

Reply:
Resolved from slack discussion